grafana-plugin/src/components/ScheduleUserDetails/ScheduleUserDetails.module.css

```diff
@@ -24,8 +24,8 @@
 }
 
 .hr {
```

Comment:
Hmm maybe we could rename it so that it's not the same name as the `HTML` element - `line-break` or `delimiter`, anything